roles/k8s_best_practices_certsuite/tasks/tests.yml

```diff
@@ -33,8 +33,8 @@
   block:
     - name: Run the Red Hat Best Practices Test Suite for Kubernetes test cases
       vars:
-        kbpc_partner_repo: "{{ (kbpc_registry | length) | ternary(kbpc_registry, 'quay.io') }}/testnetworkfunction"
-        kbpc_support_image: "{{ kbpc_support_image_name }}:{{ kbpc_support_image_version }}"
+        _kbpc_repo_org_name: "{{ (kbpc_version is version('v5.2.3', '>=') or kbpc_version == 'HEAD') | ternary('redhat-best-practices-for-k8s', 'testnetworkfunction') }}" 
```

Comment:
here's the trailing space at the end of the line that you need to remove to pass the ansible-lint check

Reply:
```suggestion
        _kbpc_repo_org_name: "{{ (kbpc_version is version('v5.2.3', '>=') or kbpc_version == 'HEAD') | ternary('redhat-best-practices-for-k8s', 'testnetworkfunction') }}"
```